app/views/items/_item_row_inventory.html.erb

```diff
@@ -1,8 +1,8 @@
 <% row_item = row.last %>
-<tr data-widget="expandable-table" aria-expanded="false">
+<tr data-widget="expandable-table" aria-expanded="false" id="item_row_inventory_<%= row_item[:item_id] %>">
```

Comment:
Why were these IDs added?

Reply:
@dorner  I added them to uniquely identify a row and check whether the item is highlighted in the feature spec. Would you rather prefer just checking in the table instead, avoiding these?